### PR TITLE
Add wrapper wdls to pipeline-tools repo

### PR DESCRIPTION
**Pull request by dshiga**
As part of repo reorganization, I'd like to move the wrapper wdls here from the skylab repo. There is an associated PR in skylab to remove the files from that repo: https://github.com/HumanCellAtlas/skylab/pull/47

### CODE REVIEW

File: adapter_pipelines/10x/adapter_example_bundle_specific.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "Wrapper10xCount.bundle_uuid": "c1677888-59fb-43bd-b3e3-05c274cdb11a",
```
Comment (rexwangcc) [inline adapter_pipelines/10x/adapter_example_bundle_specific.json]: I really love the renaming from wrapper to adapter, which is more accurate. But should we also names inside of the code to adapt to this change? I know that would be a lot more refactoring though.

File: adapter_pipelines/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+An example format map for use with submit.wdl, which requires a format map as an input.
+
+# 10x
+
+Directory containing DCP adapter for 10x pipeline.
+
+# smart_seq2
+
+Directory containing DCP adapter for Smart-seq2 pipeline.
```
Comment (rexwangcc) [inline adapter_pipelines/README.md]: I love this elaborated readme!!

File: adapter_pipelines/smart_seq2/adapter_example_static_demo.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "WrapperSs2RsemSingleSample.rsem_genome": "gs://broad-dsde-mint-dev-teststorage/demo/rsem.tar",
+  "WrapperSs2RsemSingleSample.reference_bundle": "bf51d668-3e14-4843-9bc7-5d676fdf0e01",
+  "WrapperSs2RsemSingleSample.format_map": "gs://broad-dsde-mint-dev-teststorage/format_map_example.json",
+  "WrapperSs2RsemSingleSample.submit_url": "http://api.ingest.staging.data.humancellatlas.org/",
+  "WrapperSs2RsemSingleSample.dss_url": "https://dss.staging.data.humancellatlas.org/v1",
+  "WrapperSs2RsemSingleSample.method": "Ss2RsemSingleSample",
+  "WrapperSs2RsemSingleSample.schema_version": "v3",
+  "WrapperSs2RsemSingleSample.run_type": "run",
+  "WrapperSs2RsemSingleSample.retry_seconds": 10,
+  "WrapperSs2RsemSingleSample.timeout_seconds": 600
```
Comment (rexwangcc) [inline adapter_pipelines/smart_seq2/adapter_example_static_demo.json]: @dshiga Here I guess we need an environment parameter too: `"WrapperSs2RsemSingleSample.runtime_environment": "dev"`

File: adapter_pipelines/submit.wdl [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+        -retry_seconds ${retry_seconds} \
+        -timeout_seconds ${timeout_seconds})
+
+    # Stage the files
+    files=( ${sep=' ' files} )
+    for f in "$${lb}files[@]${rb}"
+    do
+      echo "stage $f $staging_urn"
+      stage $f $staging_urn
+    done
```
Comment (rexwangcc) [inline adapter_pipelines/submit.wdl]: It seems this version of submit.wdl is not the correct version we used in out integration session? This is neither using hca-cli nor pass docker image as a parameter.

File: adapter_pipelines/submit_example.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+  "WrapperSs2RsemSingleSample.format_map": "../../hca-dcp/format_map_example.json",
+  "WrapperSs2RsemSingleSample.submit_url": "http://api.ingest.staging.data.humancellatlas.org/",
+  "WrapperSs2RsemSingleSample.method": "Ss2RsemSingleSample",
+  "WrapperSs2RsemSingleSample.schema_version": "v3",
+  "WrapperSs2RsemSingleSample.run_type": "run",
+  "WrapperSs2RsemSingleSample.retry_seconds": 10,
+  "WrapperSs2RsemSingleSample.timeout_seconds": 120
+}
```
Comment (rexwangcc) [inline adapter_pipelines/submit_example.json]: I'm wondering why this looks much shorter comparing to `https://github.com/HumanCellAtlas/skylab/blob/master/hca-dcp/submit_example.json`

File: adapter_pipelines/10x/adapter_example_bundle_specific.json (hunk 1 above)
Comment (dshiga) [reply adapter_pipelines/10x/adapter_example_bundle_specific.json]: Good idea, done

File: adapter_pipelines/smart_seq2/adapter_example_static_demo.json (hunk 3 above)
Comment (dshiga) [reply adapter_pipelines/smart_seq2/adapter_example_static_demo.json]: added

File: adapter_pipelines/submit.wdl (hunk 4 above)
Comment (dshiga) [reply adapter_pipelines/submit.wdl]: updated

File: adapter_pipelines/submit_example.json (hunk 5 above)
Comment (dshiga) [reply adapter_pipelines/submit_example.json]: fixed

**Pull request merged**